### ISSUE TITLE
Google colab demo?

### ISSUE THREAD

**Issue by ogencoglu** [gaomingqi/Track-Anything#29]
Gradio demo seems too slow. Any plans to have a Google colab demo?

**Comment by Jeffuy** [comment]
You can try[ this one](https://colab.research.google.com/gist/machinelearnear/8e8a6310f51b380d554219ca18afd287/track-anything-colab.ipynb) from Youtuber  [Machinelearnear](https://www.youtube.com/watch?v=dzscglZ_jzo) .

However, you won´t get enough VRam to run it on colab free.

**Comment by memoryunreal** [comment]
Yes, we are optimizing the RAM and VRAM usage. The google colab demo is coming soon. 

> Gradio demo seems too slow. Any plans to have a Google colab demo?

**Comment by WalidAliAlfaid** [comment]
you can use run the hugginface demo on colab if you follow this video. share the colab if it runs successfully!
https://www.youtube.com/watch?v=pOf1f-Lcu5Y&pp=ygUlMSBsaXR0bGUgY29kZXIgaHVnZ2luZyAgZmFjZSBvbiBjb2xhYg%3D%3D

**Comment by XinyueZ** [comment]
期待
